.github/workflows/releases.yml

```diff
@@ -64,7 +64,7 @@ jobs:
         with:
           name: releases
           path: dist
-      - uses: pypa/gh-action-pypi-publish@v1.6.4
```

Comment:
Ah, I think this is reverting things that have hit the mainline.

Reply:
yes, and I'm not sure why this is happening -- possibly we've reached the limits of my experience with git. When I run `git pull upstream main`, git claims everything is up to date, but clearly there's lots of recent commits on mainline that are missing from this PR. Any advice?